bool cmd
int16[] numofData
pahap/cluster[] clusterSet
---
bool test
int16[] numofData
pahap/cluster[] clusterSet
